src/main/java/co/novu/common/base/Novu.java

```diff
@@ -973,4 +983,67 @@ public UpdateOrganizationBrandResponse updateOrganizationBrand(UpdateOrganizatio
             throw e;
         }
     }
+
+    public WorkflowOverride createWorkflowOverride(CreateWorkflowOverrideRequest request) throws IOException, NovuNetworkException {
+        try {
+            return workflowOverrideHandler.createWorkflowOverride(request);
+        } catch (Exception e) {
+            log.error("Error Creating Workflow-Override", e);
+            throw e;
+        }
+    }
+
+    public BulkWorkflowOverridesResponse getWorkflowOverrides(GetWorkflowOverrideRequest request) throws IOException, NovuNetworkException {
+        try {
+            return workflowOverrideHandler.getWorkflowOverrides(request);
+        } catch (Exception e) {
+            log.error("Error Creating Workflow-Override", e);
```

Comment:
You're using the same log message for all the functions you added. Please change

Reply:
Acknowledged